plugins/arDominionB5Plugin/modules/actor/templates/_contextMenu.php

```diff
@@ -0,0 +1,41 @@
+<div class="sidebar-lowering">
```

Comment:
Not sure if we need this div now, but no need to worry ATM.

Reply:
Removed.